app/models/payment_intent.rb

```diff
@@ -9,6 +9,11 @@
 #  lock_version       :integer
 #  metadata           :text
 #  receipt_url        :string
+#  successful represents whether a payment has been received and processed by
```

Comment:
this file is autogenerated but i have no idea how to make a comment on a column in rails. if anyone has any good way to do this, lmk

Reply:
Looks like you can add a comment to the schema itself, which should get pulled to this autogenerated comment
https://blog.bigbinary.com/2016/06/21/rails-5-supports-adding-comments-migrations.html